functions/__bobthefish_colors.fish

```diff
@@ -29,6 +29,7 @@ function __bobthefish_colors -S -a color_scheme -d 'Define colors used by bobthe
       set -x color_k8s                      magenta white --bold
       set -x color_username                 white black --bold
       set -x color_hostname                 white black
+      set -x color_screen                   brgreen $colorfg
```

Comment:
can you add an example segment to `bobthefish_display_colors`, then use that to upload screenshots of how it looks in all the themes?